<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>389</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>704</width>
    <height>484</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Антиплагиат v0.0001</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>stop.png</normaloff>stop.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="text3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Порог срабатывания (%)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="percentageInput">
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>85.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="text1">
      <property name="text">
       <string>Текст 1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="text2">
      <property name="text">
       <string>Текст2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPlainTextEdit" name="leftTextField">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(39, 151, 255, 0.2);</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPlainTextEdit" name="rightTextField">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(39, 151, 255, 0.2);</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="submitButton">
      <property name="text">
       <string>Сравнить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
